Model/Config/Adminhtml/MerchantAccounts.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace Adyen\Payment\Model\Config\Adminhtml;
+
+class MerchantAccounts extends \Magento\Config\Block\System\Config\Form\Field
+{
+    protected $_template = 'Adyen_Payment::config/required_settings.phtml';
+
+    public function render(\Magento\Framework\Data\Form\Element\AbstractElement $element)
+    {
+        $element->unsScope()->unsCanUseWebsiteValue()->unsCanUseDefaultValue();
+        return parent::render($element);
+    }
+
+    protected function _getElementHtml(\Magento\Framework\Data\Form\Element\AbstractElement $element)
```

Comment:
This function seems identical to the one in the parent class, do we need to overwrite it?